### PR TITLE
qtcurve : upgrade to 1.9.0.117 (main, Qt5 and extra subports)

### PR DESCRIPTION
**Pull request by RJVB**
This brings the style up to date with upstream, supporting up to and
including Qt 5.15 . The qtcurve-extra subport now also installs the
translation files.

The GTk2 subport is updated to 1.9.0.66 .

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

OS X 10.9.5

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by kencu** [comment]
thanks!

Any idea what's going on the the CI system errors?

**Comment by RJVB** [comment]
No. TBH, I'm so used about getting linting and timeout errors that I no longer even try to find my way through the interface digging for an error in way too many logs. I see here that it says it failed to install dependencies but getting the actual failures seems to be beyond my skills?!

I did notice a few things I missed converting my own version to the one for here. Bummer, installing the translations requires a KF5 framework, somehow that detail slipped my mind :(

File: kde/qtcurve/Portfile [hunk 1]
```diff
@@ -1,25 +1,55 @@
...
+if {${subport} eq "${name}-qt5"} {
+    # Safeguard against installing a port variant for the still uncommitted KF5 ports
+    #> It should be de-commented in the mainstream port!!
+    if {![variant_isset qtonly]} {
+        default_variants +qtonly
+    }
+    #<
+
+    if {![variant_isset qtonly]} {
+        PortGroup   kf5 1.1
```
Comment (reneeotten) [inline kde/qtcurve/Portfile:24]: this is an unknown PortGroup ; perhaps you have this only locally?

File: kde/qtcurve/Portfile [hunk 2]
```diff
@@ -94,27 +116,54 @@ subport ${name}-gtk2 {
...
+        xinstall -m 644 ${filespath}/qtc_qtcurve-rjvb.themerc ${destroot}/${prefix}/share/apps/kstyle/themes/
+        xinstall -m 644 ${filespath}/qtc_qtcurve-osx.themerc ${destroot}/${prefix}/share/apps/kstyle/themes/
+        xinstall -m 644 ${filespath}/qtc_qtcurve-osxgraphite.themerc ${destroot}/${prefix}/share/apps/kstyle/themes/
+        xinstall -m 644 ${filespath}/QtCurveOSX.colors ${destroot}/${prefix}/share/apps/color-schemes/
+        xinstall -m 644 ${filespath}/OxygenOSXGraphite.colors ${destroot}/${prefix}/share/apps/color-schemes/
+    }
+    if {![variant_isset qtonly]} {
+        depends_run-append  port:ciment-icons
+        depends_build-append \
+                            port:kf5-ki18n
```
Comment (reneeotten) [inline kde/qtcurve/Portfile:144]: this dependency does not exist

**Review by reneeotten — CHANGES_REQUESTED** [review]

**Comment by pmetzger** [comment]
This has been sitting since August without any answer for @reneeotten's concerns. I'm closing it for now. It can always be reopened if there is interest.

**Pull request closed without merging**